components/filter/events.md

```diff
@@ -12,12 +12,52 @@ position: 11
 
 This article explains the available events for the Telerik Filter for Blazor:
 
+* [OnUpdate](#onupdate)
 * [ValueChanged](#valuechanged)
 
+## OnUpdate
+
+The `OnUpdate` event fires when the filter value changes. The component works directly with the bound `CompositeFilterDescriptor` value and automatically updates it when it is used with one-way binding.
+
+>caption Handle OnUpdate.
+
+````RAZOR
+@using Telerik.DataSource
+
+<TelerikFilter Value="@Value" OnUpdate="@OnFilterUpdate">
```

Comment:
This setup is minimalistic, but may be also misleading. The `Value` is not updated in any way, which may cause undesired side effects in the app. As a bare minimum, use `@bind-Value`.